assets/training/distillation/components/pipeline/spec.yaml

```diff
@@ -9,6 +9,10 @@ description: Component to generate data from teacher model enpoint and finetune
 
 inputs:
   # Compute parameters
+  instance_type_pipeline_validation:
```

Comment:
Should we update the version since we are making changes to the pipeline ?
@novaturient95

Reply:
@sharvin2187 - I think @vikasagrawal-ms has already bumped up the version of the spec as part #3290.

Reply:
Looks like the changes were published as well. I'll bump it up again, thanks!